### DISCUSSION TITLE
Add podcast audio transcripts to read later?

### DISCUSSION THREAD

**Discussion by DavisOwen** [omnivore-app/omnivore#3810] [Ideas]
Apple recently came out with native support for audio transcripts for podcasts

https://www.apple.com/newsroom/2024/03/apple-introduces-transcripts-for-apple-podcasts/

Unfortunately, there doesn't seem to be a way to "share" these yet. This feature would be very useful to me. 

I wonder if there are any low hanging fruits here? 

If we could somehow access the mp3 file in omnivore, I think we could just use some open source speech transcription library off the shelf (e.g https://github.com/exPHAT/SwiftWhisper).

**Comment by italovieira** [comment]
Related to https://github.com/omnivore-app/omnivore/issues/3736.
